CHANGELOG.md

```diff
@@ -13,9 +13,12 @@ Changelog for Cass Operator, new PRs should update the `main / unreleased` secti
 
 * [CHANGE] [#183](https://github.com/k8ssandra/cass-operator/issues/183) Move from PodDisruptionBudget v1beta1 to v1 (min. Kubernetes version 1.21)
 * [CHANGE] [#264](https://github.com/k8ssandra/cass-operator/issues/264) Generate PodTemplateSpec in CassandraDatacenter with metadata
-* [ENHANCEMENT] []() Update to Go 1.17 with updates to dependencies: Kube 1.23.4 and controller-runtime 0.11.1 
+* [FEATURE] [#309](https://github.com/k8ssandra/cass-operator/pull/309) If StatefulSets are modified in a way that they can't be updated directly, recreate them with new specs
```

Comment:
Do you really consider this a feature? :)

Reply:
Yes. It's not available in Kubernetes ;)

Reply:
Curious, outside of the bug fix what is the use case here that would be considered a feature?

Reply:
We've modified the StatefulSet before, but previously it has been protected by not making the changes if it was something else previously. I'd say this is the intended behavior of StatefulSets as of now. Recreating it is more a workaround for not being able to modify it and as such I don't think it's a bugfix.